app/js/datastore/berlin-event-data.js

```diff
@@ -0,0 +1,476 @@
+    company: 'CoinList',
+    jobTitle: 'Co-Founder and CEO',
+    twitter: '@andy_bromberg',
+    day: 'March 2',
+    time: '10:47 AM',
+    talkTitle: 'Investor Panel: Investing in a Decentralized Eco-System',
+    headshot:
+      'https://blockstack.imgix.net/d976b947-36d2-4585-9eb8-368bd97210ea_andy_bromberg_headshot.jpg',
+  },
+  {
+    name: 'Ari Paul',
+    company: 'BlockTower Capital',
+    jobTitle: 'Co-Founder and CIO',
+    twitter: '@AriDavidPaul',
+    day: 'March 2',
+    time: '10:47 AM',
+    talkTitle: 'Investor Panel: Investing in a Decentralized Eco-System',
+    headshot:
+      'https://blockstack.imgix.net/ac9f4a81-58a3-4118-9d18-bdab1f1d9993_ari+paul.png',
+  },
+  {
+    name: 'Brittany Laughlin',
+    company: 'Lattice Ventures',
+    jobTitle: 'Partner',
+    twitter: '@br_ttany',
+    day: 'March 2',
+    time: '10:47 AM',
+    talkTitle: 'Investor Panel: Investing in a Decentralized Eco-System',
+    headshot:
+      'https://blockstack.imgix.net/5909e678-a6dd-495c-93dc-1f42e7cd5705_brittany+laughlin.jpg',
+  },
+  {
+    name: 'William Mougayar',
+    company: null,
+    jobTitle: 'Author: "The Business Blockchain." Technology Futurist',
+    twitter: '@wmougayar',
+    day: 'March 2',
+    time: '10:47 AM',
+    talkTitle: 'Investor Panel: Investing in a Decentralized Eco-System',
+    headshot:
+      'https://blockstack.imgix.net/0f41ec47-d4e6-458a-9406-19c8a3ada9da_william-mougayar.jpg',
+  },
+  {
+    name: 'Richard Muirhead',
+    company: 'Fabric Ventures and OpenOcean',
+    jobTitle: 'Fabric Ventures, Founding Partner & OpenOcean, General Partner',
+    twitter: '@richardmuirhead',
+    day: 'March 2',
+    time: '10:47 AM',
+    talkTitle: 'Investor Panel: Investing in a Decentralized Eco-System',
+    headshot:
+      'https://blockstack.imgix.net/400f2ce0-144f-40e0-9336-9a33208960d0_richardmuirhead.jpg',
+  },
+  {
+    name: 'Larry Salibra',
+    company: 'Blockstack',
+    jobTitle: null,
+    twitter: '@larrysalibra',
+    day: 'March 1',
+    time: '11:05 AM',
+    talkTitle: 'Blockstack Identity & Authentication',
+    headshot:
+      'https://blockstack.imgix.net/d70de42e-5056-4cec-af87-55ab20055c5d_larry.jpg',
+  },
+  {
+    name: 'Nick Szabo',
+    company: null,
+    jobTitle: 'Blockchain, Cryptocurrency & Smart Contracts Pioneer',
+    twitter: '@NickSzabo4',
+    day: 'March 2',
+    time: '11:20 AM',
+    talkTitle: 'Social Scalability and Blockchains',
+    headshot:
+      'https://blockstack.imgix.net/66fff681-9862-4fdb-b57a-2677670d1a69_nick-szabo.jpg',
+  },
+  {
+    name: 'Jude Nelson',
+    company: 'Blockstack',
+    jobTitle: null,
+    twitter: '@JudeCNelson',
+    day: 'March 1',
+    time: '11:25 AM',
+    talkTitle: 'Subdomains',
+  },
+  {
+    name: 'Elizabeth Stark',
+    company: 'Lightning Labs',
+    jobTitle: 'Co-Founder and CEO',
+    twitter: '@starkness',
+    day: 'March 2',
+    time: '11:42 AM',
+    talkTitle: 'Building the Future of Layer 2',
```

Comment:
This was changed on the event day:

Elizabeth Stark interview with Larry Salibra.

Add me! :-D

Reply:
Oop sorry! Will add! :)